### PR TITLE
fix(host-binding): allows attributes to be removed with nullish value…

### PR DESCRIPTION
**Pull request by geometricpanda**
…s as per @HostBinding

### CODE REVIEW

File: libs/ngxtension/host-binding/src/host-binding.ts [hunk 1]
```diff
@@ -60,7 +60,13 @@ export function hostBinding<T, S extends Signal<T> | WritableSignal<T>>(
 					);
 					break;
 				case 'attr':
-					renderer.setAttribute(element, property, String(value));
+					const v = value ?? null;
+
+					if (v === null) {
```
Comment (nartc) [inline libs/ngxtension/host-binding/src/host-binding.ts:65]: nit: maybe we can just `== null` check for `value` to remove attribute if `value` is `undefined` or `null`?

Comment (geometricpanda) [reply libs/ngxtension/host-binding/src/host-binding.ts:65]: I'm not against that

**Review by nartc — APPROVED** [review]

**Comment by nartc** [comment]
@all-contributors please add @geometricpanda for code

**Comment by nartc** [comment]
Thank you for the contribution

**Pull request merged**